common/mouse.go

```diff
@@ -199,10 +199,9 @@ func (m *MouseSystem) Update(dt float32) {
 		// If the Mouse component is a tracker we always update it
 		// Check if the X-value is within range
 		// and if the Y-value is within range
-		pos := e.SpaceComponent.AABB()
 
 		if e.MouseComponent.Track || e.MouseComponent.startedDragging ||
-			mx > pos.Min.X && mx < pos.Max.X && my > pos.Min.Y && my < pos.Max.Y {
+			e.SpaceComponent.Within(engo.Point{mx, my}) {
```

Comment:
We could rename it, but it's definitely *not* a `point->AABB` reaction. `AABB` kind defines the "outer bounds" by defining  non-rotated rectangle. 

The green is what's defined by AABB, and that's *not* what's being used here. 

![](http://i.stack.imgur.com/PMNj4.png)